manticore/native/manticore.py

```diff
@@ -39,17 +39,24 @@ def __init__(self, path_or_state, argv=None, workspace_url=None, policy="random"
             initial_state = _make_initial_state(path_or_state, argv=argv, **kwargs)
         else:
             initial_state = path_or_state
-
         super().__init__(initial_state, workspace_url=workspace_url, policy=policy, **kwargs)
 
         # Move the following into a linux plugin
         self._assertions = {}
         self.trace = None
+        self.linux_machine_arch: str
```

Comment:
Since this appears on the `Manticore` object let's either prefix it with an underscore to denote that it's private or add a comment explaining what it does for RtD.